homework.js

```diff
@@ -0,0 +1,190 @@
+//Summation Function
+let arr = [];
+const summation = function(num){
+    for(i=1;i<=num;i++){
+        arr.push(i)
+    }
+    return console.log(`${arr.join(" + ")} = ${arr.reduce((a, b) => a + b, 0)}`);
+    
+}
+summation(7);
+
+//SummationEven Function
+let arrEven = [];
+const summationEven = function(num){
+    for(i=1;i<=num;i++){
+        if(i%2===0){
+        arrEven.push(i)
+        }
+    }
+    return console.log(`${arrEven.join(" + ")} = ${arrEven.reduce((a,b)=>a+b,0)}`);
+}
```

Comment:
if the initial value of the accumulator is equal to 0 we do not need to pass it, because the first array element is taken as an accumulator, and the callback will be called from the second element onwards.